docs/src/main/sphinx/security/authentication-types.rst

```diff
@@ -0,0 +1,60 @@
+====================
+Authentication types
+====================
+
+Trino supports multiple authentication types to ensure all users of the system
+are authenticated. Different authenticators allow user management in one or more
+systems. Using :doc:`TLS <tls>` is required for all authentications types.
+
+You can configure one or more authentication types with the
+``http-server.authentication.type`` property. The following authentication types
+and authenticators are available:
+
+* ``PASSSWORD`` for  :doc:`password-file`, :doc:`ldap`, and :doc:`salesforce`
+* ``OAUTH2`` for :doc:`oauth2`
+* ``CERTIFICATE`` for certificate authentication
+* ``JWT`` for Java Web Token (JWT) authentication
+* ``KERBEROS`` for :doc:`Kerberos authentication <server>`
+
+The simplest configuration to get started is password authentication backed by a
```

Comment:
to get started with

Reply:
I want to remove "simplest", I'd suggest 
> Get started with a basic password authentication configuration backed by a password file:

Reply:
Updated